src/go/RdfGoGraph.tsx

```diff
@@ -17,35 +16,19 @@ type RdfGoGraphProps = RdfGraphProps<go.DiagramEvent> & {
 
 export type RdfGoGraphDiagramRef = {
 	getDiagram(): go.Diagram | null;
-	getGraphState(): GraphState;
 };
 
 export type RdfGoGraphState = {
 	connectorNodes: Record<string, { symbolNodeId?: string; portId?: string }>;
 };
 
 export const RdfGoGraph = forwardRef<RdfGoGraphDiagramRef, RdfGoGraphProps>(function RdfGoGraph(
-	{
-		customGraphPatches,
-		initDiagram,
-		rdfPatches,
-		//onErrorCallback,
-		onGraphSelectionChanged,
-		onGraphStateChanged,
-		onSelectionChanged,
-		style,
-		symbolProvider,
-	},
+	{ graphPatches, initDiagram, onGraphSelectionChanged, onSelectionChanged, style },
```

Comment:
Trenger vi `onSelectionChanged`? Likte `onGraphSelectionChanged` mye bedre. Hvis vi kan tvinge alle klientene til å kun bruke det hadde det vært digg.

Reply:
Kan være hendig å få native UI event for en UI implementasjon. Så slipper man å "slå opp dobbelt". Er uansett ikke en obligatorisk prop.